Sources/Operators.swift

```diff
@@ -38,6 +38,21 @@ public func ~> <Service>(r: Resolver, service: Service.Type) -> Service {
     return r.resolve(service)!
 }
 
+/** Binary operator ~> equivalent to `r.resolve(Proto.Type)! as! Service`
+
+ Usage: `let mockImpl: MockImpl = r ~> DependencyA.self`
+ - Parameters:
+ - r: Resolver
+ - proto: Protocol implemented by the Service
+
+ - Returns: The resolved service type instance, retrieved via its registered protocol.
+ - Important: Fails on unresolvable service or casting error.
+ */
+infix operator ~~> : AdditionPrecedence
```

Comment:
```suggestion
infix operator ~>! : AdditionPrecedence
```

Perhaps the syntax `~>!` would make more sense?

Reply:
Thanks for your comments! I assumed this was a hard no...

I can also say that my company adopted the changes from this branch and uses this operator everywhere. We would **love** and kind of standardized solution to do this!